test/contour-grid.js

```diff
@@ -1,77 +0,0 @@
-/**
```

Comment:
The contour grid is a useful tool to troubleshoot isochrone generation when weird stuff is happening, although maybe it belongs in jsolines. It should find a home somewhere though.